src/uct/ib/mlx5/dc/dc_mlx5.h

```diff
@@ -212,6 +212,7 @@ typedef struct uct_dc_dci {
     uint8_t                       path_index; /* Path index */
     uint8_t                       next_channel_index; /* next DCI channel index
                                                          to be used by EP */
+    uint8_t                       is_shared; /* Indicates that this specific dci is shared, regardless of policy*/
```

Comment:
1. space before *
2. line seems too long
3. maybe make it uint8_t flags?

Reply:
It seems like we can't avoid that extra branch because now we don't know if dci is `UCT_DC_MLX5_EP_NO_DCI`, resulting in sanitizer failure (accessing dcis[255])